packages/integration-tests/README.md

```diff
@@ -12,6 +12,20 @@ site.
 1. Start the dev server according to the main [README instructions](/README.md#installation)
 2. In this directory, run `yarn test`.
 
+## Debugging a Test
```

Comment:
I learned a lot about how to debug Playwright tests when preparing this PR, so I recorded a Loom for future devs.